permits/templates/permits/_permit_request_actions.html

```diff
@@ -34,11 +34,31 @@ <h2>{% trans "" %}</h2>
 
       {% endif %}
 
+      {% if has_permission_to_classify and print_templates %}
+
+        <li class="nav-item">
+        <a class="btn btn-primary btn-action nav-link" id="print-tab" data-toggle="tab" href="#print" role="tab" aria-controls="print">
+            {% trans "Impression" %}
+        </a>
+        </li>
+
+      {% endif %}
+
+      {% if has_permission_to_classify and directives %}
```

Comment:
Ok, I was thinking about moving the directives into a block like this: https://github.com/yverdon/geocity/blob/main/permits/templates/permits/permit_request_detail.html#L197-L202 but maybe a tab is also fine ?